### ISSUE TITLE
Integration with virtualenvwrapper

### ISSUE THREAD

**Issue by matthew-brett** [pypa/virtualenv#1508]
Congratulations on the rewrite.

I notice that `virtualenvwrapper` doesn't integrate with the new beta:

```
$ mkvirtualenv foo
ERROR: Environment '/Users/brettmz/.virtualenvs/foo' does not contain an activate script.
```

Sure enough - I believe it is looking for a script `activate` - but the script file now depends on the shell - `activate.sh` etc. 

It would be good if this could work somehow - would it be reasonable to create / link a default `activate` script for this case?  Or should virtualenvwrapper be updated?

**Comment by gaborbernat** [comment]
This seems an oversight on our side, indeed should be just ``activate`` not ``activate.sh``.

**Comment by gaborbernat** [comment]
Resolved in #1509

**Comment by matthew-brett** [comment]
Excellent - thanks.
